samples/snippets/detect_intent_texts_with_location_test.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2017, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import os
+import uuid
+
+from detect_intent_texts_with_location import detect_intent_texts_with_location
+
+PROJECT_ID = os.getenv('GOOGLE_CLOUD_PROJECT')
+LOCATION_ID = 'europe-west2'
```

Comment:
Definitely a nit, should this also be a variable? Also seeing eu-west regions with `en-us` encoding feels odd?

Reply:
do you mean making it an environmental variable? I would prefer to also show users the format of location, as some people also get confused about that.

I updated en-US to en-GB, as europe-west2 is in London.